test/helpers.test.js

```diff
@@ -0,0 +1,22 @@
+const assert = require('assert');
+const { mapServerUri } = require('../src/helpers');
+
+describe('helpers', () => {
+  it('mapServerUri', () => {
+    const testCases = [
+      { input: null, expected: null },
+      { input: undefined, expected: undefined },
```

Comment:
do we want an error returned here?

Reply:
I thought about this, and I think the answer is no. It will be caught downstream by https, which returns a helpful message.